pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationMultiValueQueriesTest.java

```diff
@@ -519,6 +519,119 @@ public void testPercentileEst99MV() {
         .testInterSegmentAggregationResult(brokerResponse, 400000L, 0L, 800000L, 400000L, new String[]{"2147483647"});
   }
 
+  @Test
+  public void testPercentileRawEst50MV() {
+    testPercentileEstAggregationResult(50);
+  }
+
+  @Test
+  public void testPercentileRawEst90MV() {
+    testPercentileEstAggregationResult(90);
+  }
+
+  @Test
+  public void testPercentileRawEst95MV() {
+    testPercentileEstAggregationResult(95);
+  }
+
+  @Test
+  public void testPercentileRawEst99MV() {
+    testPercentileEstAggregationResult(99);
+  }
+
+  private void testPercentileEstAggregationResult(int percentile) {
+
+    Function<Serializable, String> quantileExtractor = value -> String.valueOf(ObjectSerDeUtils.QUANTILE_DIGEST_SER_DE.deserialize(BytesUtils.toBytes((String) value))
+        .getQuantile((double) percentile / 100.0));
+
+    ExpectedQueryResult<Long> expectedQueryResultsBasic =
```

Comment:
I was thinking we should modify the tests as follows:
1. Run a query with non raw version - get the result
2. Run the same query with raw version - deserialize - get the result
3. Assert that results from 1 and 2 should be exact match.
This will take away any hardcoding as well as needing to check if the results is within a % margin.

Reply:
The result might not be deterministic because of the merging order of the segments though

Reply:
Added hard coded checks for group by queries and comparison against original queries for basic query and query with filter.